pages/register.page.js

```diff
@@ -0,0 +1,109 @@
+let WebPage = require("../base/webPage");
+
+let { WebButton } = require("../elemnets/button");
+let { WebTextInput } = require("../elemnets/textInput");
+let { WebView } = require("../elemnets/view");
+
+let baseElementLocator = by.css('#account-creation_form');
+let firstNameFieldLocator = by.css('#customer_firstname');
+let lastNameFieldLocator = by.css('#customer_lastname');
+let passwordFieldLocator = by.css('#passwd');
+let addressFieldLocator = by.css('#address1');
+let cityFieldLocator = by.css('#city');
+let stateDropdownLocator = by.css('#id_state');
+let stateItemLocator = by.css('#id_state [value="10"]');
+let postcodeFieldLocator = by.css('#postcode');
+let phoneFieldLocator = by.css('#phone_mobile');
+let registerBtnLocator = by.css('#submitAccount');
+
+
+class RegisterPage extends WebPage {
+    constructor() {
+        super();
+    }
+
+    async typeFirstName(firstName) {
```

Comment:
create single method for first name and last name, add allure step there

Reply:
Done